src/index.js

```diff
@@ -0,0 +1,231 @@
+    }
+  }
+}
+
+function CapsLock() {
+  if (localStorage.capslock === null || localStorage.capslock === 'false') {
+    localStorage.setItem('capslock', true);
+    keyStorage.link[keyStorage.code.indexOf('CapsLock')].classList.add('click');
+    keyStorage.code.forEach((item, i) => {
+      if (keyStorage.link[i].classList.contains('specialButton')) {
+        keyStorage.link[i].textContent = keyStorage.link[i].textContent.toLowerCase();
+      } else {
+        keyStorage.link[i].textContent = keyStorage.link[i].textContent.toUpperCase();
+      }
+    });
+  } else {
+    localStorage.setItem('capslock', false);
+    keyStorage.link[keyStorage.code.indexOf('CapsLock')].classList.remove('click');
+    keyStorage.code.forEach((item, i) => {
+      keyStorage.link[i].textContent = keyStorage.link[i].textContent.toLowerCase();
+    });
+  }
+}
+
+function Shift() {
+  keyStorage.shiftArray.length = 0;
+  keyStorage.link.forEach((_item, i) => {
+    if (keyStorage.link[i].classList.contains('firstLine')) {
+      keyStorage.shiftArray.push(keyStorage.link[i]);
+      keyStorage.firstline.push(keyStorage.link[i].textContent);
+    }
+    if (localStorage.capslock === 'false' && !(keyStorage.link[i].classList.contains('specialButton'))) {
+      keyStorage.link[i].textContent = keyStorage.link[i].textContent.toUpperCase();
+    }
+    if (localStorage.capslock === 'true' && !(keyStorage.link[i].classList.contains('specialButton'))) {
+      keyStorage.link[i].textContent = keyStorage.link[i].textContent.toLowerCase();
+    }
+  });
+  keyStorage.shiftArray.forEach((item, i) => {
+    keyStorage.shiftArray[i].textContent = keyStorage.firstlineS[i];
+  });
+}
+
+function ChangeLanguage() {
+  if (localStorage.lang === null || localStorage.lang === 'ru') {
+    localStorage.setItem('lang', 'en');
+    keyStorage.code.forEach((item, i) => {
+      keyStorage.link[i].textContent = keyStorage.en[i];
+      if (localStorage.capslock === 'true' && !(keyStorage.link[i].classList.contains('specialButton'))) {
+        keyStorage.link[i].textContent = keyStorage.link[i].textContent.toUpperCase();
+      }
+    });
+    if (keyStorage.firstline.length !== 0) {
+      keyStorage.firstline[0] = '~';
+    }
+  } else {
+    localStorage.setItem('lang', 'ru');
+    keyStorage.code.forEach((item, i) => {
+      keyStorage.link[i].textContent = keyStorage.ru[i];
+      if (localStorage.capslock === 'true' && !(keyStorage.link[i].classList.contains('specialButton'))) {
+        keyStorage.link[i].textContent = keyStorage.link[i].textContent.toUpperCase();
+      }
+    });
+    if (keyStorage.firstline.length !== 0) {
+      keyStorage.firstline[0] = 'ё';
+    }
+  }
+}
+
+for (let i = 0; i < 63; i += 1) {
+  const button = new Box(keyStorage.ru[i], keyStorage.en[i], keyStorage.code[i]);
+  button.setText();
+  ul.appendChild(button.elem);
+  button.elem.classList.add('button', keyStorage.code[i].toLocaleLowerCase());
+  if (button.elem.textContent === 'tab'
```

Comment:
As I can see, you always compare button.elem.textContent, so I suggest the "switch" statement.

Reply:
Done